src/Configurator.php

```diff
@@ -2,34 +2,135 @@
 
 namespace TomPHP\ConfigServiceProvider;
 
-interface Configurator
+use TomPHP\ConfigServiceProvider\FileReader\FileLocator;
+use TomPHP\ConfigServiceProvider\FileReader\ReaderFactory;
+use TomPHP\ConfigServiceProvider\Exception\NoMatchingFilesException;
+use TomPHP\ConfigServiceProvider\Exception\UnknownSettingException;
```

Comment:
If there were more I'd agree but I kinda prefer them imported in this case. Am happy either way though really.